subsys/logging/Kconfig.template.log_format_config

```diff
@@ -7,6 +7,7 @@ choice "LOG_BACKEND_$(backend)_OUTPUT"
 
 config LOG_BACKEND_$(backend)_OUTPUT_TEXT
 	bool "Text"
+	select LOG_OUTPUT
```

Comment:
Commit is not valid as it differs from upstream. Should be updated to fromtree, and patches must match 1:1

Reply:
updated to fromtree

Reply:
however, it does not match 1:1 because in upstream there is one more file that was changed but this file do not exist in sdk-zephyr (it came after the upmerge). What is the approach in that case?

Reply:
Pull in the commits that are needed for this to apply cleanly, otherwise you can't bring it in as fromlist/fromtree and you would need to noup it

Reply:
done, had to fetch one more commit.

Reply:
@nordicjm can you take another look so we could merge it soon?